README.md

```diff
@@ -0,0 +1,13 @@
+# IJoke (PAC1)
+
+## Why?
+
+We want to provide a convenient, modern-looking web application that allows users to read, add, and like jokes.
+
+## What?
+
```

Comment:
Move "what" to the paragraph above since it answers both questions "Why" and "What"